.ci/blc-website-preview

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
Are all of these just copies of the ones from `ambassador-docs.git` ?

Reply:
Yes